Table/Columns.cc

```diff
@@ -52,6 +52,22 @@ std::unique_ptr<Column> Column::FromField(const pugi::xml_node& field) {
     return column;
 }
 
+std::unique_ptr<Column> Column::FromValues(const std::vector<string>& values, string name) {
+    unique_ptr<Column> column;
+
+    column = make_unique<DataColumn<string>>(name);
+    column->Resize(values.size());
+    for (auto row_index = 0; row_index < values.size(); row_index++) {
+        (dynamic_cast<DataColumn<string>*>(column.get()))->SetFromValue(values[row_index], row_index);
```

Comment:
should just do the dynamic cast one, before the loop, and should also check that it is valid after casting